packages/@azure/servicebus/data-plane/lib/queueClient.ts

```diff
@@ -128,41 +123,40 @@ export class QueueClient extends Client {
    * of the message.
    * For more information please see {@link https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-partitioning#use-of-partition-keys Use of partition keys}
    *
-   * @param data - Message to send.  Will be sent as UTF8-encoded JSON string.
-   * @returns Promise<Delivery>
+   * @param data - Message to send.
+   * @returns Promise<void>
    */
-  async send(data: SendableMessageInfo): Promise<Delivery> {
+  async send(data: SendableMessageInfo): Promise<void> {
     const sender = MessageSender.create(this._context);
     return sender.send(data);
```

Comment:
If return type is void then why are we returning data here?

Reply:
`return sender.send(data)` doesnt return the data.
It returns the Promise returned by `sender.send()` which if you look at MessageSender.ts resolves to void.